app/views/api_docs/pages/home.md

```diff
@@ -0,0 +1,55 @@
+This is documentation for the Department for Education (DfE)’s postgraduate
```

Comment:
Jesse and Paul may have views on design / copy?

Reply:
I was going to get these out there and then invite them to comment.